File: arrays/reverse-string.js
//Create a function that reverses a string:

function reverse(str) {
  //check input
  if(!str || str.length < 2 || typeof str !== 'string') {
    return 'no good'
  }
  splitString = str.split("")
  reverseList = []
  stringLength = str.length - 1

  for(let i=stringLength; i>=0; i--) {
    reverseList.push(splitString[i])
  }
  return reverseList.join('')
}

const reverse2 = (str) => str.split('').reverse().join('')


console.log(reverse('jokagdsf'))
console.log(reverse2('jokagdsf'))

File: arrays/reverse-string.py
#Create a function that reverses a string:

def reverse(string):
    #check type
    if len(string) < 2 or type(string) != str:
        return 'not good'

    reverseStr = ''
    strLen = len(string)-1

    for i in range(strLen, -1, -1):
        reverseStr += string[i]

    return reverseStr

def reverse2(string):
    return string[::-1]

print(reverse('test'))
print(reverse2('test'))
